apps/src/templates/multiple-sections-assigner.module.scss

```diff
@@ -13,40 +25,35 @@
   }
 }
 
-.sectionList {
-  margin: 24px 0 12px 0;
-  padding: 0 0 24px 0;
-  border-bottom: 1px solid $neutral_dark50;
-
-  h5 {
-    margin-bottom: 0;
-  }
-
-  .sectionListOptionsContainer {
-    &:focus {
-      outline: 2px solid blue;  
-      outline-offset: 2px;     
-  }
+  .sectionList {
+    margin: 24px 0 12px 0;
+    padding: 0 0 5px 5px;
   
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: space-between;
-    margin: 12px 0 4px 0;
-    overflow-y: scroll;
-    max-height: 300px;
-
-    label {
-      width: 280px;
-      margin-bottom: 8px;
+    h5 {
+      margin-bottom: 0;
+    }
+  
+    .sectionListOptionsContainer {
+      display: flex;
+      flex-wrap: wrap;
+      justify-content: space-between;
+      margin: 12px 0 4px 0;
+  
+      label {
+        width: 280px;
+        margin-bottom: 8px;
+      }
     }
   }
-}
-
-.modalHeader h3 {
-  margin-bottom: 0;
-}
+  
```

Comment:
Tiny nit: I don't think we need the indentation here and we shouldn't add spaces to empty lines.

Reply:
Weird, I was kind of expecting the linter to let me know, but it looks like it didn't catch it - I can make that change!  Good catch!